package.json

```diff
@@ -8,5 +8,16 @@
   },
   "keywords": [],
   "author": "",
+<<<<<<< HEAD
```

Comment:
Ah I must have misread the git gui terminal before (sorry I should have asked for it to be zoomed in!) can you revert any changes to this file and delete the package-lock.json file which is at the root of the repo please? :)

Reply:
Think I've done this now!